webdriver-spec.html

```diff
@@ -699,13 +699,18 @@
 <section>
 <h2>Interface</h2>
 
-<pre class=idl>partial interface Navigator {
-	[Unforgeable] readonly attribute boolean webdriver;
+<p>If the <dfn>webdriver-active flag</dfn> is true,
```

Comment:
OK, well in that case I think this is fine, could tweak this in the future if needed.